src/tests/ftest/deployment/basic_checkout.py

```diff
@@ -20,16 +19,45 @@ class BasicCheckout(IorTestBase, MdtestBase):
     :avocado: recursive
     """
 
-    def test_basic_checkout(self):
+    def test_basiccheckout_sanity(self):
         """
         Test Description: Bundles four tests into one and run in the
                           following sequence - ior_small, mdtest_small,
                           ec_smoke and autotest.
         :avocado: tags=all,deployment,full_regression
         :avocado: tags=hw,large
         :avocado: tags=dfuse,ior,mdtest
-        :avocado: tags=basiccheckout
+        :avocado: tags=basiccheckout,basiccheckout_sanity
         """
+
+        # ior easy
+        self.run_performance_ior(namespace="/run/ior_dfs_sx/*", use_stonewalling_read=True)
+        self.run_performance_ior(namespace="/run/ior_dfs_ec_16p2gx/*", use_stonewalling_read=True)
+
+        # mdtest easy
+        self.run_performance_mdtest(namespace="/run/mdtest_dfs_s1/*")
+        self.run_performance_mdtest(namespace="/run/mdtest_dfs_ec_16p2g1/*")
```

Comment:
In CI, HW Large doesn't have enough nodes for EC_16P2. At most, you can use EC_8P2 in CI